app/components/Exchange/Exchange.jsx

```diff
@@ -316,23 +316,21 @@ class Exchange extends React.Component {
         return this.state.feeStatus[asset.get("id")] && this.state.feeStatus[asset.get("id")].fee;
     }
 
-    _verifyFee(fee, sellAmount, sellBalance, coreBalance) {
+    _verifyFee(fee, sell, sellBalance, coreBalance) {
         let coreFee = this._getFee();
 
-        let sellSum = fee.getAmount() + sellAmount;
-        if (fee.asset_id === "1.3.0") {
+        if (fee.asset_id === "1.3.0") { // Very strange check
             if (coreFee.getAmount() <= coreBalance) {
                 return "1.3.0";
             } else {
                 return null;
             }
         } else {
+            let sellSum = (sell.asset_id == fee.asset_id) ? (fee.getAmount() + sell.getAmount()) : sell.getAmount();
```

Comment:
Please use strict equivalences (===) since that's what's used everywhere else in the code.

Reply:
ok, i'll change it to strict eq